src/components/Dialogs/ImageInfo/ImageInfoDialogComponent.scss

```diff
@@ -0,0 +1,38 @@
+.image-info-dialog {
+    min-width: 60%;
+    height: 100%;
+    user-select: none;
+
+    .bp3-dialog-body {
+        height: 100%;
+        overflow: hidden;
+        margin-top: 10px;
+        margin-left: 25px;
+
+        .bp3-input-group {
```

Comment:
is this used?

Reply:
No, it is not used. This is also a legacy from FileBrowserComponent. I will remove it.